core/classifiers/TextClassifier/TextClassifier.py

```diff
@@ -78,7 +78,7 @@ def train(self, training_data):
         for answer_group in training_data:
             for answer in answer_group['answers']:
                 x.append(answer)
-                y.append(str(answer_group['answer_group_index']))
+                y.append(answer_group['answer_group_index'])
```

Comment:
Good find, @GanitGenius.

Reply:
@seanlip now we know why was the trained model data used in frontend tests for TextClassifier had the [issue](https://github.com/oppia/oppia/pull/4573#discussion_r167125036). Because @anmolshkl was converting answer group index to strings before training and hence trained model also had those classes in string format.

Reply:
Ah, that's a good catch, @GanitGenius! Thanks.